corehq/apps/users/static/users/js/edit_commcare_user.js

```diff
@@ -0,0 +1,113 @@
+$(function () {
```

Comment:
So many things begging to be refactored in here... Good for you for being disciplined and only doing one thing at a time. I was about to suggest `hqDefine` given the size of this document ready handler, but it looks like that's more than a simple endeavor.